fixtures/v2.xml

```diff
@@ -1621,8 +1621,8 @@
         </tasks>
       </taskmgr>
       <server>
-        <boot-time>2016-09-06T22:52:15Z</boot-time>
-        <current-time>2016-09-07T03:15:44Z</current-time>
+        <boot-time>2021-07-15T05:11:08.926Z</boot-time>
+        <current-time>2021-07-15T10:25:39.396Z</current-time>
```

Comment:
Not sure why you're modifying unrelated fixtures. Please revert this.

Reply:
I'm sorry for the confusion. Tests for v3 depend on v2 (e08083dd245eca4c38844e121b291ccf8fe296f3). So I had to either use older timestamps in v3 fixtures or update v2 fixtures to keep tests passing.

Do you like to have this reverted anyway?

Reply:
Ok, that makes sense.